views/home.jade

```diff
@@ -30,19 +30,14 @@ block content
   |              <div class="col-lg-8 col-lg-offset-2 text-center">
   |                  <h2 class="section-heading">What we do</h2>
   |                  <hr class="light">
-  |                  <p class="text-faded">GREETINGS! Welcome to Employee Survey.
+  |                  <p class="text-faded"> Welcome to Employee Survey.
```

Comment:
I'm surprised you haven't noticed the company name change yet. It's kind of everywhere